phpstan.neon

```diff
@@ -0,0 +1,12 @@
+parameters:
+  level: 3
```

Comment:
oh yeah, this feels way too low to be truly useful

Reply:
Now 6.

Reply:
One has to start somewhere! ;)